app/Http/Controllers/Api/NotesController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Events\NoteAdded;
+use App\Helpers\Helper;
+use App\Http\Controllers\Controller;
+use App\Models\Asset;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+class NotesController extends Controller
+{
+    public function store(Request $request)
+    {
+        $item = Asset::findOrFail($request->input("id"));
+        $this->authorize('update', $item);
+
+        $validated = $request->validate([
+            'note' => 'required|string|max:500']);
```

Comment:
Can we move this to the top of the method and add `type` is `required` and has to be `asset`? It's not super necessary for this PR but will make it clearer for future expansion...

It'll look something like 

```php 
$validated = $request->validate([
    'note' => 'required|string|max:500',
    'type' => [
        'required',
        Rule::in(['asset']),
    ],
]);
```

https://laravel.com/docs/10.x/validation#rule-in